Files/MultilingualResources/Files.es-ES.xlf

```diff
@@ -612,9 +612,9 @@
         </trans-unit>
         <trans-unit id="SettingsPreferencesTerminalApplicationsSampleProfiles.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
           <source>View sample profiles</source>
-          <target state="translated">Ver perfiles personalizados</target>
+          <target state="new">View sample profiles</target>
```

Comment:
Why was this reverted?

Reply:
Hmmm, may be some conflicts

Reply:
@tsvietOK Except for this, everything else looks good to go...

Reply:
@yaichenbaum I fixed this in 688c03937aa2825ca286a9ab81c5f44b062d5dec